database/interfaces.go

```diff
@@ -2,481 +2,481 @@ package database
 
 import (
 	"context"
-	"github.com/momentum-xyz/ubercontroller/pkg/cmath"
 
 	"github.com/google/uuid"
 
+	"github.com/momentum-xyz/ubercontroller/pkg/cmath"
 	"github.com/momentum-xyz/ubercontroller/types/entry"
 	"github.com/momentum-xyz/ubercontroller/utils/modify"
 )
 
 type DB interface {
 	CommonDB
```

Comment:
I think we need to do the same with CommonDB, because DB itself is something like upper level for all DBs
for example, we can have on DB level some methods that requires to operate with several DBs (node seeding?)

Reply:
Added method GetCommonDB()